### DISCUSSION TITLE
Bromite F-Droid

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1984] [General]
Release in Github Latest Version.
Bromite
v100.0.4896.92
(13 hours ago)
Still not available in F-Droid Bromite Repo.
Please Release in F-Droid.

**Comment by ghost** [comment]
Is this issue or author still not release in repo.
Any one facing same?

**Comment by csagan5** [comment]
It is in F-Droid now.

**Reply by ghost** [reply]
Thanks
